File: video/src/main/java/com/leory/badminton/video/mvp/ui/VideoMainActivity.java
package com.leory.badminton.video.mvp.ui;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.leory.badminton.video.R;
import com.leory.commonlib.base.BaseActivity;
import com.leory.commonlib.di.component.AppComponent;
import com.leory.interactions.RouterHub;

/**
 * Describe : 视频主页
 * Author : zhuly
 * Date : 2018-06-14
 */


public class VideoMainActivity extends BaseActivity {



    @Override
    public void setupActivityComponent(@NonNull AppComponent appComponent) {

    }

    @Override
    public int initView(@Nullable Bundle savedInstanceState) {
        return R.layout.activity_video_main;
    }

    @Override
    public void initData(@Nullable Bundle savedInstanceState) {

    }

}

File: video/src/main/java/com/leory/badminton/video/mvp/ui/VideoMainActivity.kt
package com.leory.badminton.video.mvp.ui

import android.os.Bundle

import com.leory.badminton.video.R
import com.leory.commonlib.base.BaseActivity
import com.leory.commonlib.mvp.IPresenter

/**
 * Describe : 视频主页
 * Author : zhuly
 * Date : 2018-06-14
 */


class VideoMainActivity : BaseActivity<IPresenter>() {


    override fun initView(savedInstanceState: Bundle?): Int = R.layout.activity_video_main


    override fun initData(savedInstanceState: Bundle?) {

    }

}
